File: Python_Programming/Day_3/program14.py

# Description : Write a program that accepts number from the user & returns 
#               factorial of the given number.

def Factorial(No):
    Fact = 1
    
    for i in range(1 , No+1 , 1):
        Fact = Fact * i

    return Fact

def main():

    Value = int(input("Enter the number : "))

    Ret = Factorial(Value)

    print("Factorial is : ",Ret)

if __name__ == "__main__":
    main()

File: C_Programming/Day_3/program14.c
// Description : Write a program that accepts number from the user & returns 
//               factorial of the given number.

// Description : Write a program to check whether the number is perfect number or not.

# include<stdio.h>

int Factorial(int iNo)
{
    int iCnt = 0;
    int iFact = 1;
    for(iCnt = 1 ; iCnt <= iNo ; iCnt++)
    {
        iFact = iFact * iCnt;
    }

    return iFact;

}


int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter the number : ");
    scanf("%d",&iValue);

    iRet = Factorial(iValue);

    printf("Factorial is : %d\n",iRet);

    return 0;
}

